src/folding/nova/circuits.rs

```diff
@@ -545,10 +553,7 @@ mod tests {
         let cs = ConstraintSystem::<Fr>::new_ref();
 
         // prepare the circuit to obtain its R1CS
-        let test_F_circuit_dummy = TestFCircuit::<Fr> {
-            z_i: Fr::zero(),
-            z_i1: Fr::zero(),
-        };
+        let F_circuit = TestFCircuit::<Fr> { _f: PhantomData };
```

Comment:
```suggestion
        let big_f_circuit = TestFCircuit::<Fr> { _f: PhantomData };
```

I'd avoid variable names that start with capital letters

Reply:
I'm not sure in cases like this. I don't have a strong opinion, but I will write the logic that I followed to see if it makes sense:

When implementing a paper I try to keep names the most similar to the paper, so the paper itself can be used as documentation/reference of the code. This helps future readers of the code, but also for maintainers itself to avoid confusions (and mistakes).

Additionally in this case, using `big_f_circuit` here for the F function circuit would mean that the Augmented F (currently `augmented_F_circuit`) would need to be `augmented_big_f_circuit`, but the `big` word does not bring any relevant information rather than enlarging the variable name (for example there is no `small_F` to distinct from), just that the paper uses `F` to refer to the function.

There are other values that use uppercase in the paper, in some cases is to distinct them from the lowercase ones. For example `u_i` and `U_i` following the paper's notation, use uppercase to distinct two different elements using the same letter, but they might look confusing if we put `u_i, big_u_i` instead of `u_i, U_i` as they appear in the paper, since the uppercase one is not the bigger version from `u_i`, but the "uppercase" version, thus maybe we could write it as `u_i, uppercase_u_i`, but for doing that I think that it makes more sense to directly use `u_i, U_i` and keeping it closer to the paper notation.

Reply:
Then what about calling it `f_circuit` but without the capital? Would that work?

Reply:
But if we apply this criteria to `F_circuit` to be `f_circuit`, which criteria do we apply to `u_i, U_i`?

Reply:
Ohhh you got me there. 

Let's leave it as is then!